cmake/setup_custom_targets.cmake

```diff
@@ -140,6 +153,9 @@ FILE(WRITE ${CMAKE_BINARY_DIR}${CMAKE_FILES_DIRECTORY}/print_info.cmake
 #
 #    indent         - indent all headers and source file
 #
+#    relocate       - (only available on Darwin machines) fix RPATH for external 
+#                     libraries, if packaging was requested
+#
```

Comment:
You could try to output this only on Mac and if available, by splitting the whole thing into more than one MESSAGE statement.